src/core/or/sendme.c

```diff
@@ -286,10 +286,37 @@ send_circuit_level_sendme(circuit_t *circ, crypt_path_t *layer_hint,
   return 0;
 }
 
+/* Put the crypto.b_digest in the sendme_digest. */
+static void
+note_cell_digest(const relay_crypto_t *crypto)
```

Comment:
"Note" isn't quite right here -- let's try "record"?

Reply:
(This should probably go into the relay_crypto module)

Reply:
Moved it to `relay_crypto.c` and named `relay_crypto_record_sendme_digest()`

See new commit 4d6d633132